transform/snowflake-dbt/models/hightouch/blapi/schema.yml

```diff
@@ -0,0 +1,225 @@
+      - name: updated_at
+        description: Date that the subscription was updated at BLAPI.
+      - name: invoice_number
+        description: The invoice number from BLAPI.
+      - name: stripe_charge_id
+        description: Stripe's charge ID, taken from BLAPI.
+      - name: num_seats
+        description: Number of seats purchased.
+      - name: sku
+        description: The SKU of the subscription purchased.
+      - name: pricebookentryid
+        description: ID of price book entry, taken from BLAPI.
+      - name: actual_renewal_date
+        description: The date that the renewal happened (?), taken from Stripe
+      - name: renewal_start_date
+        description: Renewal start date. Always 1 day after actual renewal date.
+      - name: renewal_end_date
+        description: Renewal end date. Always 1 year after actual renewal date.
+      - name: sfdc_migrated_opportunity_sfid
+        description: Migrated opportunity ID from Salesforce.
+      - name: is_renewed
+        description: >
+          Boolean flag to indicate if there was a previous subscription. If yes, then this record is considered a
+          renewal and value is true.
+      - name: renewed_from_total
+      - name: license_key
+        description: The id of the license if not null. Otherwise the subscription ID.
+      - name: purchase_order_num
+      - name: line1
+        description: Line 1 of address from BLAPI address.
+      - name: line2
+        description: Line 2 of address from BLAPI address.
+      - name: street_address
+        description: Line 1 and line 2 of address from BLAPI address joined in a single string.
+      - name: postal_code
+        description: Postal codefrom BLAPI address.
+      - name: city
+        description: City from BLAPI address.
+      - name: state
+        description: State from BLAPI address.
+      - name: country
+        description: Country name from postal code mapping or country from BLAPI address.
+      - name: state_code
+        description: State code from postal code mapping or state from BLAPI address.
+      - name: country_code
+        description: Country code from postal code mapping or country from BLAPI address.
+      - name: hightouch_sync_eligible
+        description: >
+          Boolean flag used for filtering which rows to sync with hightouch. True if subscription was updated at 
+          2021-08-18 or later.
+      - name: account_external_id
+        description: The Salesforce ID of the customer's contact or the custommer's account.
+      - name: account_type
+        description: The account's type, as taken from Salesforce account.
+      - name: account_sfid
+        description: The account's Salesforce ID, as taken from Salesforce account.
+      - name: contact_external_id
+      - name: contact_sfid
+        description: Salesforce ID of the contact for the Salesforce account related to this customer.
+      - name: opportunity_external_id
```

Comment:
This field is calculated in dbt models and synced to salesforce, acts as foreign key to join with salesforce data that is synced back to snowflake. To identify if the current row already exists in salesforce.

Reply:
opportunity_external_id